app/views/lead_providers/guidance/ecf_usage.md

```diff
@@ -0,0 +1,71 @@
+<h2 class="app-contents-list__title">Contents</h2>
+
+<ol class="app-contents-list__list">
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent"><a class="govuk-link app-contents-list__link" href="#continuing-the-registration-process">Continuing the registration process</a></li>
+  <li class="app-contents-list__list-item app-contents-list__list-item--parent"><a class="govuk-link app-contents-list__link" href="#declaring-that-a-participant-has-started-their-induction">Declaring that a participant has started their induction</a></li>
+</ol>
+
+<hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6">
```

Comment:
Why the mix of html and md?

Reply:
couldn't get components to work with my very simple markdown renderer... so gave up

Reply:
Ehh, want some govspeak? 😂 

In all seriousness though, should these files be erb rather than markdown?